### ISSUE TITLE
the details of YOLOV3

### ISSUE THREAD

**Issue by Jwy-jump** [prasshantg/odla_data#3]
can you show me the detials about how to running of yolov3? the unsupported layers?

**Comment by prasshantg** [comment]
@JIAWENYANG we used Darknet application for it and unsupported layers were running on cpu

**Comment by Jwy-jump** [comment]
But the three part of .nvdla can not run on the nvdla vp?I don't know how to use it?thanks.

**Comment by prasshantg** [comment]
I added some custom split layers for it and then linked these 3 parts in Darknet config, see https://github.com/prasshantg/darknet/blob/master/cfg/yolov3-odla.cfg

**Comment by Jwy-jump** [comment]
thanks a lot

**Comment by Jwy-jump** [comment]
I also want to get the tools which can convert loadable to .fbuf，because I want to konw more about achieving Yolov3 by nvDla,thanks.

**Comment by prasshantg** [comment]
loadable is in fbuf format only. loadable is just name we call to that fbuf file.

**Comment by Jwy-jump** [comment]
![image](https://user-images.githubusercontent.com/16818951/70438504-f2f2c800-1ac8-11ea-8c62-7bdb3740bd5b.png)
Hi,can you help me about this picture? @prasshantg

**Comment by prasshantg** [comment]
can you please share .cfg?

**Comment by JIAWENYANG** [comment]
[net]
# Testing
batch=1
subdivisions=1
# Training
# batch=64
# subdivisions=2
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches = 500200
policy=steps
steps=400000,450000
scales=.1,.1

[odla]
instance=0
loadable=yolov3-tiny.fbuf

[split]
input_layer=-1
tensor=1

[yolo]
mask = 3,4,5
anchors = 10,14,  23,27,  37,58,  81,82,  135,169,  344,319
classes=80
num=6
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1

[split]
input_layer=-3
tensor=0

[yolo]
mask = 1,2,3
anchors = 10,14,  23,27,  37,58,  81,82,  135,169,  344,319
classes=80
num=6
jitter=.3
ignore_thresh = .7
truth_thresh = 1
random=1

**Comment by Jwy-jump** [comment]
I run this on the nvdla VP

**Comment by Jwy-jump** [comment]
![捕获](https://user-images.githubusercontent.com/16818951/70499138-73f6a180-1b53-11ea-808f-fc943054b4cd.PNG)

**Comment by Jwy-jump** [comment]
![image](https://user-images.githubusercontent.com/16818951/70693568-aab9ec80-1cf8-11ea-8ed1-3334f2568fd6.png)
I am sorry to bother you,I use your config on the NVDLA VP

**Comment by Jwy-jump** [comment]
![捕获](https://user-images.githubusercontent.com/16818951/70693741-07b5a280-1cf9-11ea-9c1e-dbcc97a72063.PNG)
@ @prasshantg
